website/cue/reference/components/sources/host_metrics.cue

```diff
@@ -116,6 +117,72 @@ components: sources: host_metrics: {
 				unit:    "seconds"
 			}
 		}
+		cgroups: {
+			common:      false
+			description: #"Options for the "cgroups" metrics collector. This collector is only available on Linux systems, as the cgroups (control groups) system is only present on Linux."#
+			required:    false
+			type: object: options: {
+				base: {
+					common:      false
+					required:    false
+					description: "The base cgroup name to provide metrics for"
+					type: string: {
+						default: null
+						examples: ["/", "system.slice/snapd.service"]
+						syntax: "literal"
+					}
+				}
+				groups: {
+					common:      false
+					required:    false
+					description: "Lists of device name patterns to include or exclude."
+					type: object: options: {
+						includes: {
+							required: false
+							common:   false
+							description: """
+								The list of cgroup name patterns for which to gather metrics.
+								Defaults to including all cgroups.
+								The patterns are matched using [globbing](#globbing).
+								"""
+							type: array: {
+								default: ["*"]
+								items: type: string: {
+									examples: ["user.slice/*", "*.service"]
+									syntax: "literal"
+								}
+							}
+						}
+						excludes: {
+							required: false
+							common:   false
+							description: """
+								The list of cgroup name patterns for which to gather metrics.
+								Defaults to excluding no cgroups.
+								The patterns are matched using [globbing](#globbing).
+								"""
+							type: array: {
+								default: []
+								items: type: string: {
+									examples: ["user.slice/*", "*.service"]
+									syntax: "literal"
+								}
+							}
+						}
+					}
+				}
+				levels: {
+					common:      false
+					required:    false
+					description: "The number of levels of the cgroups hierarchy for which to report metrics. A value of `1` means just the root or named cgroup."
+					type: uint: {
+						unit:    null
+						default: 3
```

Comment:
I had copied that from `systemd-cgtop`, thinking that it might be similar in other tools. I see now, however, that nothing else has the same limit of three levels, so I am thinking either `0` (no recursion, only the named group) or `u64::MAX` (full recursion) would be more appropriate. Any opinions on which way to go on this?